addons/csw/CfgWeapons.hpp

```diff
@@ -0,0 +1,250 @@
+class CfgWeapons {
+    class Launcher;
+    class Launcher_Base_F: Launcher {
+        class WeaponSlotsInfo;
+    };
+    class GVAR(m3CarryTripod): Launcher_Base_F {
+        class GVAR(cswOptions) {
```

Comment:
I'd name that class just options - the GVAR macro would resolve to ace_csw_cswOptions, kind of redundant.